src/reducers/ui.js

```diff
@@ -24,8 +24,23 @@ function addNotification(state, type, severity, payload = {}) {
   );
 }
 
+export function reduceRoot(rootState, action) {
+  return rootState.update('ui', (state) => {
+    switch (action.type) {
+      case 'GIST_EXPORT_NOT_DISPLAYED':
```

Comment:
That’s a good thought. Right now notifications are only ever removed by a click on the close button in the notification, but there are situations in which it makes sense to remove a notification about an unsuccessful action that was later done successfully, including the one you bring up here. Another example would be login errors. I don’t think we want to clear _all_ notifications in these scenarios, but we should clear out the relevant one(s).

That said, as written this has the same behavior as the status quo, and I’m inclined to make an overall auto-clear-error-notification change separately. In fact that seems like the kind of good-for-beginner change I’m anxious to accumulate in advance of Hack Day : )